suggestionBox/suggestionBox-library/src/main/java/org/silverpeas/components/suggestionbox/model/SuggestionCollection.java

```diff
@@ -281,7 +280,7 @@ public Suggestion publish(final Suggestion suggestion) {
     // Persisting the publishing.
     final SuggestionRepository suggestionRepository = getSuggestionRepository();
     Pair<Suggestion, Boolean> result =
-        Transaction.performInOne(new Transaction.Process<Pair<Suggestion, Boolean>>() {
+        Transaction.performInOne(new Process<Pair<Suggestion, Boolean>>() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1604)